pyiron_gpl/pacemaker/pacemaker.py

```diff
@@ -0,0 +1,231 @@
+        res_dict["rmse_energy_low"] = ef_rmses[:, 1]
+        res_dict["rmse_forces"] = ef_rmses[:, 2]
+        res_dict["rmse_forces_low"] = ef_rmses[:, 3]
+        return res_dict
+
+    def collect_output(self):
+        output_potential_filename = self.get_output_potential_filename()
+        final_potential_filename = self.get_final_potential_filename()
+
+        copyfile(output_potential_filename, final_potential_filename)
+        with open(output_potential_filename, "r") as f:
+            yaml_lines = f.readlines()
+        final_potential_yaml_string = "".join(yaml_lines)
+
+        bbasis = ACEBBasisSet(output_potential_filename)
+        cbasis = bbasis.to_ACECTildeBasisSet()
+
+        cbasis.save(self.get_final_potential_filename_ace())
+        with open(self.get_final_potential_filename_ace(), "r") as f:
+            ace_lines = f.readlines()
+        final_potential_ace_string = "".join(ace_lines)
+
+        elements_name = bbasis.elements_name
+
+        with self.project_hdf5.open("output/potential") as h5out:
+            h5out["yaml"] = final_potential_yaml_string
+            h5out["ace"] = final_potential_ace_string
+            h5out["elements_name"] = elements_name
+
+        log_res_dict = self._analyse_log()
+
+        with self.project_hdf5.open("output/log") as h5out:
+            for key, arr in log_res_dict.items():
+                h5out[key] = arr
+
+    def get_lammps_potential(self):
+        elements_name = self["output/potential/elements_name"]
+        elem = " ".join(elements_name)
+        pot_file_name = self.get_final_potential_filename_ace()
+        pot_dict = {
+            'Config': [["pair_style pace\n", "pair_coeff  * * {} {}\n".format(pot_file_name, elem)]],
+            'Filename': [""],
+            'Model': ["ACE"],
+            'Name': [self.job_name],
+            'Species': [elements_name]
+        }
+
+        ace_potential = pd.DataFrame(pot_dict)
+
+        return ace_potential
+
+    def to_hdf(self, hdf=None, group_name=None):
+        super().to_hdf(
+            hdf=hdf,
+            group_name=group_name
+        )
+        with self.project_hdf5.open("input") as h5in:
+            self.input.to_hdf(h5in)
+
+    def from_hdf(self, hdf=None, group_name=None):
+        super().from_hdf(
+            hdf=hdf,
+            group_name=group_name
+        )
+        with self.project_hdf5.open("input") as h5in:
+            self.input.from_hdf(h5in)
+
+    def get_output_potential_filename(self):
+        return os.path.join(self.working_directory, "output_potential.yaml")
+
+    def get_final_potential_filename(self):
+        return os.path.join(self.working_directory, self.job_name+".yaml")
+
+    def get_final_potential_filename_ace(self):
+        return os.path.join(self.working_directory, self.job_name+".ace")
+
+    def get_current_potential_filename(self):
+        return os.path.join(self.working_directory, "interim_potential_1.yaml")
+
+    def get_current_potential(self):
```

Comment:
Decorate with `@import_alarm` and can remove if/else clause (sim. `get_final_potential`)

Reply:
done